.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -335,7 +335,7 @@ jobs:
               if [[ $keys_in_use == $exp_nb_keys ]]; then
                 break
               fi
-              echo "Waiting until key rotation starts (seeing $keys_in_use keys)"
+              echo "Waiting until key rotation completes (seeing $keys_in_use keys, expected $exp_nb_keys)"
```

Comment:
It looks like `starts` was changed into `completes` by mistake here.

Reply:
Oopsie, fixed.